sdk/src/types/block/address/bech32.rs

```diff
@@ -88,8 +58,8 @@ impl Packable for Hrp {
 
     #[inline]
     fn pack<P: Packer>(&self, packer: &mut P) -> Result<(), P::Error> {
-        self.len.pack(packer)?;
-        packer.pack_bytes(&self.inner[..self.len as usize])?;
+        (self.0.len() as u8).pack(packer)?;
+        packer.pack_bytes(&self.0.byte_iter().collect::<Vec<_>>())?;
```

Comment:
That's unfortunate, do they really not expose the bytes without an iterator?

Reply:
If not, please add a TODO to revisit later, we need to ask them to add one

Reply:
they dont expose it.

Reply:
@Alex6323 Can you open an issue (or a PR?) on their repository please? And add a TODO in the meantime

Reply:
okay

Reply:
I'll do it.

Reply:
Opened a PR.